zproject/default_settings.py

```diff
@@ -421,6 +421,10 @@
 # testing.
 USING_PGROONGA = False
 
+# Whether the server is using the pg_search (paradedb) extension
+# for full-text search.
+USING_PG_SEARCH = False
```

Comment:
I wonder if the right naming scheme for this is `USING_PARADEDB` or `USING_PG_SEARCH`; maybe a question for upstream as to how they prefer to be referred to in documentation, which is the most prominent place this might appear.

Reply:
I would use USING_PARADEDB. We will rename our extension to paradedb eventually.